src/pynq/pynq_driver.cc

```diff
@@ -126,6 +127,10 @@ class VTADevice {
     VTAWriteMappedReg(vta_compute_handle_, 0x0, VTA_AUTORESTART);
     VTAWriteMappedReg(vta_store_handle_, 0x0, VTA_AUTORESTART);
 
+    // Allow device to respond
```

Comment:
Could you add more explanation here to make sure that this sleep is well understood for posterity (the summarized comment by @remotego should suffice)